tests/group_ops.rs

```diff
@@ -154,6 +154,66 @@ fn group_rotate_private_key_non_admin() -> Result<(), IronOxideErr> {
     Ok(())
 }
 
+#[test]
+fn rotate_all() -> Result<(), IronOxideErr> {
+    use ironoxide::{user::UserCreateOpts, InitAndRotationCheck};
+    let account_id: UserId = create_id_all_classes("").try_into()?;
+    let jwt = gen_jwt(Some(account_id.id())).0;
+    IronOxide::user_create(&jwt, USER_PASSWORD, &UserCreateOpts::new(true))?;
+    let device = IronOxide::generate_new_device(
+        &gen_jwt(Some(account_id.id())).0,
+        USER_PASSWORD,
+        &Default::default(),
+    )?;
+    let creator_sdk = ironoxide::initialize(&device)?;
+    // making non-default groups so I can specify needs_rotation of true
+    let group_create1 = creator_sdk.group_create(&GroupCreateOpts::new(
```

Comment:
`let group_create1 = GroupCreateOpts { needs_rotation: true, ..Default::default() };`

Reply:
Sadly this won't work because the fields of the struct are private